src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -230,6 +230,11 @@ This option enables a one-step build for both your Android project and Flutter p
 This option is convenient when you work on both parts simultaneously and rapidly
 iterate, but your team must install the Flutter SDK to build the host app.
 
+{{site.alert.tip}}
+By default the host app is the `:app` Gradle project. To change the name of this project, 
```

Comment:
Do you mean "the host app **is listed** in the..."? The "host app IS the gradle project" confuses me.

Also, since we want a clarification anyway:

- Add a comma after "By default".
- Indent each line in the note by 2 spaces.
- add "file" to the end of the file. =>. "in the Flutter module's `grade.properties` file.

Thanks!!

Reply:
Done! PTAL